src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const errorMessage = 'Bzzz... Error!';
+
+  return (firstArg) => {
+    return (secondArg) => {
```

Comment:
I don't really like the idea of using first, second, third etc.

I'd rarther use 'x' and 'y'